src/jit/codegen.h

```diff
@@ -750,7 +750,12 @@ protected :
             struct
             {
                 regNumberSmall  scRegNum;
-                regNumberSmall  scOtherReg; // used for "other half" of long var
+                
+                // Used for:
+                //  - "other half" of long var on OSs with 32 bit size registers - x86.
+                //  - for System V structs it stores the second register 
+                //    used to pass a register passed struct.
```

Comment:
Are we going to have an issue when we get to doing x86 SystemV structs, where we'll need both uses?